Describe the DataGrid - export to pdf demo (#188)
DevExpress/devextreme-demos: JSDemos/Demos/DataGrid/ExportToPDF/description.md

> Our [DataGrid](/Documentation/ApiReference/UI_Widgets/dxDataGrid/) allows you to easily and accurately export its contents to a PDF file. To enable the export operation, you must reference or import <a href="https://github.com/MrRio/jsPDF" target="_blank">jsPDF</a> library.

Reviewer: 'easily and accurately' - taken from Ray's description? 

I think you need an article here: "the jsPDF library". It differs when the usage below, where you don't add the word "library".

Author: @Abadzhev Hello again, I attached the screenshot with a similar description rewritten by Ray.

![image](https://user-images.githubusercontent.com/33040934/95055183-99185a00-06fb-11eb-8cad-3369cfb3667a.png)

Reviewer: Okay - not going to argue with that :-)

> Our [DataGrid](/Documentation/ApiReference/UI_Widgets/dxDataGrid/) allows you to easily and accurately export its contents to a PDF file. To enable the export operation, you must reference or import the <a href="https://github.com/MrRio/jsPDF" target="_blank">jsPDF</a> library.

Reviewer 2: ```suggestion
Our [DataGrid](/Documentation/ApiReference/UI_Widgets/dxDataGrid/) allows you to easily and accurately export its contents to PDF. To enable the export operation, you must reference or import the <a href="https://github.com/MrRio/jsPDF" target="_blank">jsPDF</a> library.
```

> Once you’ve referenced/imported `jsPDF`, call the **DevExpress**.**pdfExporter**.**exportDataGrid(options)** method to export **DataGrid** content in the PDF format.

Reviewer 3: ```suggestion
Once you've referenced/imported `jsPDF`, call the **DevExpress**.**pdfExporter**.**exportDataGrid(options)** method to export **DataGrid** content in the PDF format.
```

> Once you've referenced/imported `jsPDF`, call the **DevExpress**.**pdfExporter**.**exportDataGrid(options)** method to export **DataGrid** content in the PDF format.

Reviewer 2: ```suggestion
Once you've referenced/imported `jsPDF`, call the **DevExpress**.**pdfExporter**.**exportDataGrid(options)** method to export **DataGrid** content to PDF.
```

> Review the export code in the button's [onClick](/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#onClick) handler to learn more.

Reviewer 3: ```suggestion
In this demo, the **exportDataGrid(options)** method is called when you click the Export to PDF button. Review the export code in the button's [onClick](/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#onClick) handler to learn more.
```
